adminSiteClient/ChartEditorPage.tsx

```diff
@@ -206,6 +273,7 @@ export class ChartEditorPage
     }
 
     dispose!: IReactionDisposer
+    disposeDetailHandler!: IReactionDisposer
```

Comment:
One mobx pain is to have to write these disposers... a slight improvement might be to add them to an array, we use this pattern in other places: https://github.com/owid/owid-grapher/blob/f7cd315cc8ac99c2c3dbafae6d2450f9e673dbb6/grapher/core/Grapher.tsx#L917-L948